decidim-comments/lib/decidim/comments/commentable.rb

```diff
@@ -38,6 +37,13 @@ def comments_have_votes?
         def commentable_type
           self.class.name
         end
+
+        # Public: Defines which users will receive a notification when a comment is created.
+        # This method can be overridden at each resource model to include or exclude
+        # other users, eg. admins.
```

Comment:
Please add docs on the method signature: should it return an Array of IDs? Or an Array of `Decidim::User`?